.github/workflows/main.yml

```
@@ -0,0 +1,17 @@
+name: CI
+on:
+  - push
+  - pull_request
+jobs:
+  lint:
+    name: Run awesome-lint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '18'
```

Might as well keep this evergreen:
```suggestion
          node-version: 'latest'
```

---

Yup. That’s a very good idea.